custom/icds_reports/ucr/data_sources/ag_care_monthly.json

```diff
@@ -0,0 +1,715 @@
+                "type": "root_doc",
+                "expression": {
+                  "datatype": "date",
+                  "type": "property_name",
+                  "property_name": "closed_on"
+                }
+              }
+            },
+            "property_value": 0,
+            "comment": null
+          }
+        ]
+      }
+    },
+    "configured_filter": {
+      "operator": "eq",
+      "type": "boolean_expression",
+      "expression": {
+        "type": "property_name",
+        "property_name": "type",
+        "datatype": null
+      },
+      "property_value": "ag_care",
+      "comment": null
+    },
+    "validations": [
+      {
+        "name": "owned_by_awc",
+        "error_message": "Case not owned by AWC",
+        "expression": {
+          "type": "boolean_expression",
+          "operator": "eq",
+          "expression": {
+            "location_id_expression": {
+              "type": "property_name",
+              "property_name": "owner_id"
+            },
+            "type": "location_type_name"
+          },
+          "property_value": "awc"
+        }
+      }
+    ],
+    "configured_indicators": [
+      {
+        "display_name": null,
+        "datatype": "string",
+        "expression": {
+          "expression": {
+            "datatype": null,
+            "type": "property_name",
+            "property_name": "owner_id"
+          },
+          "type": "root_doc"
+        },
+        "is_primary_key": false,
+        "transform": {},
+        "is_nullable": false,
+        "type": "expression",
+        "column_id": "awc_id",
+        "create_index": false,
+        "comment": null
+      },
+      {
+        "display_name": null,
+        "datatype": "string",
+        "type": "expression",
+        "is_primary_key": true,
+        "transform": {},
+        "is_nullable": false,
+        "expression": {
+          "location_id": {
+            "expression": {
+              "datatype": null,
+              "type": "property_name",
+              "property_name": "owner_id"
+            },
+            "type": "root_doc"
+          },
+          "location_type": "supervisor",
+          "location_property": "_id",
+          "type": "ancestor_location"
+        },
+        "column_id": "supervisor_id",
+        "create_index": false,
+        "comment": null
+      },
+      {
+        "display_name": null,
+        "datatype": "string",
+        "type": "expression",
+        "is_primary_key": false,
+        "transform": {},
+        "is_nullable": true,
+        "expression": {
+          "location_id": {
+            "expression": {
+              "datatype": null,
+              "type": "property_name",
+              "property_name": "owner_id"
+            },
+            "type": "root_doc"
+          },
+          "location_type": "block",
+          "location_property": "_id",
+          "type": "ancestor_location"
+        },
+        "column_id": "block_id",
```

Comment:
are these higher level location columns necessary

Reply:
Akshita/Sunaina can you respond?